TheWarWithin/Priorities/MageFrost.simc

```diff
@@ -29,12 +32,28 @@ actions.aoe+=/cone_of_cold,if=target.within10&!talent.coldest_snap&active_enemie
 actions.aoe+=/frostbolt
 actions.aoe+=/call_action_list,name=movement
 
+actions.aoe+=/cone_of_cold,if=talent.coldest_snap&cooldown.frozen_orb.remains>4&(prev_gcd.1.comet_storm|prev_gcd.1.frozen_orb&!talent.comet_storm|cooldown.comet_storm.remains>15&!talent.frostfire_bolt)
+actions.aoe+=/frozen_orb,if=cooldown_react&((!prev_gcd.1.cone_of_cold|!talent.isothermic_core)&(!prev_gcd.1.glacial_spike|!freezable))
+actions.aoe+=/blizzard,if=!prev_gcd.1.glacial_spike|!freezable
```

Comment:
We should think about the different prev/prev_gcd entries some more, though this doesn't keep us from moving forward with this PR.  There can be a difference between "the last GCD spell was X" vs. "X was used <= 1 GCD ago" (or if you want to relax the requirement in case somebody hits another GCD spell.

Reply:
I can definitely revist on the next pass. Sometimes it's because you need to spellqueue (anytime flurry is involved) which would be fine for time since, sometimes it's just "this spell must be after that spell".